packages/core/src/reporters/mutation-test-report-helper.ts

```diff
@@ -233,7 +233,7 @@ export class MutationTestReportHelper {
   }
 
   private toMutantResult(mutantResult: MutantResult, remapTestIds: (ids: string[] | undefined) => string[] | undefined): schema.MutantResult {
-    const { fileName, location, killedBy, coveredBy, ...apiMutant } = mutantResult;
+    const { fileName: _, location, killedBy, coveredBy, ...apiMutant } = mutantResult;
```

Comment:
I understand why we do this, but it feels a bit strange. Maybe its better to use [`ignoreRestSiblings`](https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings)?